docs/user_guides/primitives_pipelines/primitives/TadGAN.rst

```diff
@@ -15,21 +15,18 @@ argument                    type                description
 **parameters**
 ------------------------------------------------------------------------------------------------------------------------------------------------
  ``X``                      ``numpy.ndarray``   n-dimensional array containing the input sequences for the model 
-
+ ``y``                      ``numpy.ndarray``   n-dimensional array containing the target sequences we want to reconstruct
```

Comment:
Is ``y`` always the same as ``X``? If yes, we can add some quick explanations here.

Reply:
`y` is a selected channel of `X`. So it can be `X` or a few channels from `X`. I can add that to the documentation.